presets/4.3/vtx/Avatar_HD.txt

```diff
@@ -0,0 +1,49 @@
+#$ TITLE: Avatar HD VTXs
+#$ FIRMWARE_VERSION: 4.2
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: VTX
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: vtx, vtx table, Avatar, Walksnail, digital, whoop
+#$ AUTHOR: jappyjan
+
+#$ PARSER: MARKED
+
+#$ DESCRIPTION: 
+#$ DESCRIPTION: <img src="https://walksnail.help/uploads/images/system/2022-09/WS-logo.png" height="90px" style="object-fit: cover; margin-left: auto; margin-right: auto; display: block;"/>
+#$ DESCRIPTION: 
+#$ DESCRIPTION: # MSP Setup
+#$ DESCRIPTION: ## for all Walksnail VTX's
+
+#$ OPTION BEGIN (UNCHECKED): use displayport on uart 1
+serial 0 1 115200 57600 0 115200
+set displayport_msp_serial = 0
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): use displayport on uart 2
+serial 1 1 115200 57600 0 115200
+set displayport_msp_serial = 1
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): use displayport on uart 3
+serial 2 1 115200 57600 0 115200
+set displayport_msp_serial = 2
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): use displayport on uart 4
+serial 3 1 115200 57600 0 115200
+set displayport_msp_serial = 3
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED): use displayport on uart 5
+serial 4 1 115200 57600 0 115200
```

Comment:
what this command is doing exactly? Does it work in 4.3 and 4.2?

Reply:
here is the definition of that command.
https://github.com/betaflight/betaflight/blob/master/docs/Serial.md

and yes, as far as i know it works just fine on both BF versions